drizzlepac/haputils/mvm_quality_graphics.py

```diff
@@ -8,44 +8,36 @@
 PUBLIC INTERFACE FOR THIS MODULE
  - build_mvm_plots(HDF5_FILE, output_basename='', display_plot=False):
 
-pthon mvm_quality_graphics.py mvm_qa_dataframe.h5
+python mvm_quality_graphics.py mvm_qa_dataframe.h5
 
 """
 
 # Standard library imports
 import argparse
```

Comment:
General comment: THANK YOU.  In my rush to get the base version of mvm_quality_graphics.py out before I had a vacation day, I forgot to do "Pep" updates.

Reply:
Glad to do it!